### DISCUSSION TITLE
stacked points (scatterplot) spidering

### DISCUSSION THREAD

**Discussion by verres** [visgl/deck.gl#5370] [Q&A]
We are using deck.gl for a project mapping a large dataset that has many different datapoints that share the very same latitude/longitude data, thus 'stack' on top of each other when rendered.   We'd like a means to 'spider them' when a user hovers over the point - so they are independently selectable -- basically as described her **https://carto.com/blog/stacking-chips-a-map-hack/

is there a way to do this easily with the scatterplot of deck.gl?

Thanks

**Comment by Pessimistress** [comment]
You can use the [METER_OFFSETS](https://deck.gl/docs/developer-guide/coordinate-systems#supported-coordinate-systems) coordinate system to arrange objects relative to a geolocation.

Full example: https://codepen.io/Pessimistress/pen/dypEWMO

**Reply by verres** [reply]
thanks!
